testproject/Assets/Tests/Runtime/MultiprocessRuntime/Helpers/BuildMultiprocessTestPlayer.cs

```diff
@@ -6,91 +6,118 @@
 #endif
 using UnityEngine;
 
-/// <summary>
-/// This is needed as Unity throws "An abnormal situation has occurred: the PlayerLoop internal function has been called recursively. Please contact Customer Support with a sample project so that we can reproduce the problem and troubleshoot it."
-/// when trying to build from Setup() steps in tests.
-/// </summary>
-public static class BuildMultiprocessTestPlayer
+namespace MLAPI.MultiprocessRuntimeTests
 {
-    public const string MultiprocessBaseMenuName = "MLAPI/Multiprocess Test";
-    public const string BuildAndExecuteMenuName = MultiprocessBaseMenuName + "/Build Test Player #t";
-    public const string MainSceneName = "MultiprocessTestScene";
-    private static string BuildPathDirectory => Path.Combine(Path.GetDirectoryName(Application.dataPath), "Builds", "MultiprocessTests");
-    public static string BuildPath => Path.Combine(BuildPathDirectory, "MultiprocessTestPlayer");
+    /// <summary>
+    /// This is needed as Unity throws "An abnormal situation has occurred: the PlayerLoop internal function has been called recursively. Please contact Customer Support with a sample project so that we can reproduce the problem and troubleshoot it."
+    /// when trying to build from Setup() steps in tests.
+    /// </summary>
+    public static class BuildMultiprocessTestPlayer
+    {
+        public const string MultiprocessBaseMenuName = "MLAPI/Multiprocess Test";
+        public const string BuildAndExecuteMenuName = MultiprocessBaseMenuName + "/Build Test Player #t";
+        public const string MainSceneName = "MultiprocessTestScene";
+
+        public const string BuildInfoFileName = "buildInfo.txt";
```

Comment:
I think this default value would make more sense we'll parse it as a JSON file :)
```suggestion
        public const string BuildInfoFileName = "BuildInfo.json";
```